src/hooks/use-ads.tsx

```diff
@@ -0,0 +1,60 @@
+import type { MutableRefObject } from 'react';
+import { useEffect } from 'react';
+
+const SCRIPT_URL
+  = '//cdn.carbonads.com/carbon.js?serve=CEAI627Y&placement=sequelizeorg';
+
+type OnEnvironment = 'mobile' | 'desktop' | 'all';
+
+type InitProps = {
+  ref?: MutableRefObject<HTMLInputElement | undefined>,
+  selector?: string,
+  on?: OnEnvironment,
+};
+
+function shouldRender(on: OnEnvironment) {
+  const isMobile = window.matchMedia('(max-width: 996px)').matches;
```

Comment:
Consider using a hook for this that will listen to the change event, such as https://github.com/streamich/react-use/blob/master/docs/useMedia.md

Reply:
I integrated react-hookz now